docs/docs/References/Plugin_Operations.md

```diff
@@ -190,6 +190,7 @@ Argument | Type | Description
 direct_source | [DirectSource](Classes.md#directsource) | The source associated with this operation.
 repository | [RepositoryDefinition](Schemas_and_Autogenerated_Classes.md#repositorydefinition-class) | The repository associated with this source.
 source_config | [SourceConfigDefinition](Schemas_and_Autogenerated_Classes.md#sourceconfigdefinition-class) | The source config associated with this source.
+optional_snapshot_parameters | [SnapshotParametersDefinition](Classes.md#snapshotparametersdefinition) | **Optional.** The snapshot parameters. The value is `None` when executed during a snapshot policy.
```

Comment:
Should we remove the "optional" part? We need this parameter to be part of the method definition, but it can be `None`